enumerable.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
```

Comment:
#### `REQUIRED`

:bug: There's no return value. Also, let's handle cases where a block is given.

```ruby
irb(main):001:0> require_relative 'enumerable'
=> true
irb(main):002:0> [1,2,3,4,5].each{|i| i > 2}
=> [1, 2, 3, 4, 5]
irb(main):003:0> [1,2,3,4,5].my_each{|i| i > 2}
=> nil
```

#### Do this for all other methods. :blush:

Reply:
Hi, @bolah2009. Thanks for pointing me the right address. The methods were corrected and compared with the built-in methods. I am awaiting further revision. 

Regards!
**Andres**